cityofaustin/atd-vz-data
# Update crash details data tables to be compatible with new data model

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/17451

## Testing
**URL to test:** 

**Steps to test:**
1. Follow instructions in https://github.com/cityofaustin/atd-vz-data/pull/1440 to start the data model collector branch and run the CRIS import ETL, or if you already have it set up use that!
2. Apply the hasura migrations and metadata from this branch
3. Go to a crash page, see that the data tables are now pulling from the new `GET_CRASH` query. We are also now using all the new lookup tables.
4. Test editing the fields that are editable.
5. Test updating injuries for a person, see that the data is refetched and the Injuries data table card updates automatically.

---
#### Ship list
